sleap/gui/color.py

```diff
@@ -19,14 +19,15 @@
 from sleap.instance import Instance, Track, Node
 from sleap.io.dataset import Labels
 from sleap.prefs import prefs
+from sleap.io.cameras import InstanceGroup, RecordingSession
```

Comment:
Nit: python imports should be sorted alphabetically
```suggestion
from sleap.io.cameras import InstanceGroup, RecordingSession
from sleap.io.dataset import Labels
from sleap.prefs import prefs
```